$NetBSD: patch-src___tags,v 1.1 2019/08/24 11:07:34 jaapb Exp $

Remove dependency on deriving, do not build syntax package (patch from upstream)
--- src/_tags.orig	2019-02-08 11:31:12.000000000 +0000
+++ src/_tags
@@ -1,4 +1,4 @@
-<{lib,tools,ocamlbuild,syntax}/**/*>:warn(+A-4-6-7-9-27-37-40-42-44-48)
+<{lib,tools,ocamlbuild}/**/*>:warn(+A-4-6-7-9-27-37-40-42-44-48)
 true:keep_locs
 
 <lib/type_dir/*.ml{,i}>:eliom_ppx,thread
@@ -23,7 +23,7 @@ true:keep_locs
 
 <lib/client/*.ml{,i}>: eliom_ppx
 <lib/client/*.ml{,i}>:package(ocsigenserver.cookies,ocsigenserver.polytables,ocsigenserver.baselib.base)
-<lib/client/*.ml{,i}>:package(deriving.runtime,js_of_ocaml.deriving)
+<lib/client/*.ml{,i}>:package(js_of_ocaml.deriving)
 <lib/client/*.ml{,i}>:package(lwt_react,tyxml.functor,js_of_ocaml.tyxml,js_of_ocaml-lwt,js_of_ocaml-lwt.logger)
 <lib/client/*.ml{,i}>:package(react,js_of_ocaml,reactiveData)
 
@@ -34,9 +34,6 @@ true:keep_locs
 <lib/server/monitor/*.ml{,i}>:package(lwt,ocsigenserver,ocsigenserver.ext,tyxml)
 <lib/server/monitor/*.ml{,i}>:I(src/lib/server)
 
-<syntax/pa_*.*>: syntax(camlp4o),package(camlp4.quotations.o,camlp4.extend,bytes)
-<syntax/pa_*.ml{,i}>: I(+camlp4/Camlp4Parsers)
-
 <ppx/**/*>: package(ppx_tools, compiler-libs.common, ppx_tools.metaquot)
 
 <ocamlbuild/ocamlbuild_eliom.ml{,i}>: package(ocamlbuild,js_of_ocaml.ocamlbuild)
